src/bot/requirements.txt

```diff
@@ -8,4 +8,5 @@ pytz
 aioredis
 python-docx
 cryptography==3.4.8
-pymorphy2
\ No newline at end of file
+pymorphy2
+pytest
```

Comment:
Could you please create a separate file requirements-dev.txt ? Where will be the following content:
```
-r requirements.txt
pytest
```
It will allow to build prod bot docker image without dev dependencies.

Reply:
done 👍